### PR TITLE
chore: release 3.0.0

### PR DESCRIPTION
**Pull request by jooola**
I propose to release 3.0.0 the ~2022-10-13~ 2022-10-10 (Thursday).

I there anything missing that should be shipped in 3.0.0 ?

### CODE REVIEW

**Comment by paddatrapper** [comment]
This is a good idea - was going to suggest the same this week. I don't think there anything missing that has to be in 3.0.0

**Comment by jooola** [comment]
We could also release today, I was just pushing it to Thursday to almost make a full month from the first beta. But there aren't any urgent fixes.

**Comment by paddatrapper** [comment]
I'm happy to release today

**Comment by jooola** [comment]
Hmm, latest tests on bionic does log some warnings. I'll fix this quickly

**Review by paddatrapper — APPROVED** [review]

File: docs/releases/3.0.0.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+The following issues may need a workaround for the time being. Please search the [issues](https://github.com/libretime/libretime/issues) before reporting problems not listed below.
+
+### Missing AAC encoder for Liquidsoap
+
+AAC streams aren't working out of the box because the [current distributions packages of Liquidsoap are lacking a AAC encoder](https://github.com/libretime/libretime/issues/2184). If you need AAC streams, you need to [install Liquidsoap differently](https://www.liquidsoap.info/doc-1.4.4/install.html).
+
+### On Ubuntu Bionic, analyzing some FLAC files fails
+
+[On Ubuntu Bionic, analyzing some FLAC files fails with `UnplayableFileError`](https://github.com/libretime/libretime/issues/2218) because of an upstream bug in Liquidsoap. If you encounter this, you should [upgrade to a more recent distribution version](../developer-manual/development/releases.md#distributions-releases-support).
```
[the remark this answers is not in the document]
Comment (paddatrapper) [reply docs/releases/3.0.0.md]: Yup that looks good

**Pull request merged**